### ISSUE TITLE
md-autocomplete: suggestions cannot display with keyboard navigation for variable height elements

### ISSUE THREAD

**Issue by du6** [angular/material#10534]
**Actual Behavior**:
 -  `What is the issue? *`
md-autocomplete suggestions cannot display with keyboard navigation for variable height elements
 -  `What is the expected behavior?`
expect to fully display the suggestion element with ARROW DOWN clicked

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:`
http://codepen.io/anon/pen/Wpgzpa
 -  `Details:`
It can be reproduced by simply increase the height of one suggestion element in the md-virtual-repeat-container:
https://screenshot.googleplex.com/SEXKEB10OPX

**Comment by Splaktar** [comment]
I am not able to access the https://screenshot.googleplex.com link.

However, `md-autocomplete` uses `md-virtual-repeat` under the covers. As mentioned in https://github.com/angular/material/issues/9422#issuecomment-350540226, Virtual Repeat does not support dynamically sized items and thus neither does `md-autocomplete`.
